www/content/workstation/knife_bootstrap.md

```diff
@@ -519,3 +519,15 @@ knife bootstrap --bootstrap-wget-options "-e use_proxy=yes -e http://myproxy.com
 ``` bash
 knife bootstrap --bootstrap-install-command "curl -l http://mycustomserver.com/custom_install_chef_script.sh | sudo bash -s --"
 ```
+
+**Bootstrap a Windows node via WinRM using a run list and environment**
+
+``` bash
+knife bootstrap -o winrm FQDN_OR_IP -U username -P 'password' --node-name NODE_NAME --run-list 'recipe[cookbook]' -E ENV_NAME
```

Comment:
For consistency, could you use addresses/names like the other examples? 
It might also be worth having one of these show the short form for specifying protocol and credentials. 

`knife bootstrap winrm://user:PASSWORD@192.0.2.0 ... `

Reply:
Updated examples